src/analysis/special_functions/trigonometric/angle.lean

```diff
@@ -82,6 +82,66 @@ by simp [←coe_int_mul_eq_zsmul]
 @[simp] lemma coe_pi_add_coe_pi : (π : real.angle) + π = 0 :=
 by rw [←two_nsmul, two_nsmul_coe_pi]
 
+lemma two_nsmul_coe_eq_iff {ψ θ : ℝ} :
+  (2 : ℕ) • (ψ : angle) = (2 : ℕ) • (θ : angle) ↔ ((ψ : angle) = θ ∨ (ψ : angle) = θ + π) :=
```

Comment:
As this lemma is syntactically weaker than `two_nsmul_eq_iff`, I think it would make sense to just inline the proof there. Do you agree?

Reply:
Done.